recurring_ical_events.py

```diff
@@ -360,6 +360,10 @@ def __init__(self, calendar, keep_recurrence_attributes=False):
         self.repetitions = []
         for event in calendar.walk("VEVENT"):
             self.repetitions.append(RepeatedEvent(event, keep_recurrence_attributes))
+        for event in calendar.walk("VTODO"):
+            self.repetitions.append(RepeatedTodo(event, keep_recurrence_attributes))
+        for event in calendar.walk("VJOURNAL"):
```

Comment:
Does `walk(("VEVENT", "VTODO"))` work? - not a blocker for merge

Reply:
Oh, did I really commit this?  It's not part of the "broken tests" and it's anyway broken.  I'll move it over to a separate pull request.

Reply:
> Does `walk(("VEVENT", "VTODO"))` work? - not a blocker for merge

Apparently not:

```
(Pdb) self.walk(('VEVENT', 'VTODO'))
*** AttributeError: 'tuple' object has no attribute 'upper'
```